bundles/http/Citrix/Canvas-LMS/script.js

```diff
@@ -0,0 +1,950 @@
+const moment = library.load('moment-timezone');
+
+const startDate = moment.utc().subtract(30, 'd').format();
+const endDate = moment.utc().add(30, 'd').format();
```

Comment:
These variables doesn't need to accessed by multiple functions there is not requirement to have global variables, we can define them inside `canvasFullSync` function

Reply:
We modified the script  accordingly by verifying the need for these two variables